pipeline/runtime/runtest.go

```diff
@@ -67,10 +67,10 @@ func executeRunTestStep(ctx context.Context, engine *engine.Engine, r *api.Start
 		err = collectionErr
 	}
 
-	exportEnvs := fetchExportedVarsFromEnvFile(exportEnvFile, out)
+	exportEnvs, _ := fetchExportedVarsFromEnvFile(exportEnvFile, out)
 	if len(r.OutputVars) > 0 {
 		if exited != nil && exited.Exited && exited.ExitCode == 0 {
-			outputs := fetchExportedVarsFromEnvFile(outputFile, out)
+			outputs, err := fetchExportedVarsFromEnvFile(outputFile, out) //nolint:govet
```

Comment:
Do we have to add this code here
if len(r.OutputVars) > 0 {
			// only return err when output vars are expected
			return exited, outputs, exportEnvs, err
		}

Reply:
If I don't add it then it will fail for plugins. Since plugin can or cannot export output vars.

Reply:
So only failing when len(r.OutputVars) > 0 but we got error while parsing outputs.

Reply:
So if you asked for runtest step, we already have a check `if len(r.OutputVars) > 0` here.